main.py

```diff
@@ -171,6 +171,12 @@ def run_scenario(current_scenario):
         is_save_data=True,
     )
     current_scenario.mpl.compute_test_score()
+    # if satcking clear the partners' model (in order to compute well the time of the methods used after the computation of the federated score)
+    if current_scenario.multi_partner_learning_approach == "stacking": 
+        for partner in current_scenario.partners_list:
+            partner.model=None 
```

Comment:
There is no `model` attribute to the `Partner` object. Do you target `model_weights` here?

Reply:
I have the feeling that this action would be better located at the end of `compute_test_score()`, don't you think?
(Rationale behind is to try to keep only high level functions in `main.py`)

Reply:
typo in the comment: "satcking" => stacking"